vm-docker/Dockerfile

```diff
@@ -0,0 +1,117 @@
+FROM 'google/cloud-sdk:slim'
+
+# install packages 
+RUN apt-get update && apt-get install -y \
+    apt-utils \
+    default-jdk \
+    g++ \
+    less \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    pkg-config \
+    python3-venv \
+    unzip \
+    wget \
+    zlib1g-dev \
+    && \
+	# clean up apt cache
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir home/tools 
```

Comment:
/home/tools seems like an unconventional location since 'tools' isn't a user account. 
`/tools` or `/usr/tools` or `/home/root/tools` might be better.

Reply:
Thanks Ben, I didnt think of this! I'm going with `/tools` for now but do not have a strong preference.